### DISCUSSION TITLE
Update of presets

### DISCUSSION THREAD

**Discussion by Bundy01** [wwmm/easyeffects#2139] [General]
Hi,

I wonder about the presets after making a [repository](https://github.com/Bundy01/EasyEffects-Presets). If I understand correctly, the `.json` files are modified according to the versions of EasyEffects.

Are these changes done on major versions (7,8...) or even on minor versions (7.0.1,7.0.2...) ?

How to reset the presets without recreating everything ?

If it is not possible, I suggest to be able to do it with the `save` presets button.

Regards.

**Comment by wwmm** [comment]
>Are these changes done on major versions (7,8...) or even on minor versions (7.0.1,7.0.2...) ?

I only change the preset format when there is really no other option. Usually when a major feature that require API changes is implemented. In the past this happened when we started to support microphone effects and more recently when support for multiple filters were added.

In some of these cases some people of the community kindly provided websites that allowed presets to be updated to a newer version. And in EE `7.0.0` built-in support for `6.x.x` presets were added. But it is not always easy to do this.

>How to reset the presets without recreating everything ?

There isn't a standard procedure. Ideally I would like to never change the preset format again.

**Comment by Bundy01** [comment]
Thank you.

>There isn't a standard procedure. Ideally I would like to never change the preset format again.

In case of change, how to be informed (changelog or others)?

**Reply by wwmm** [reply]
>In case of change, how to be informed (changelog or others)?

Usually when we have to change presets formats we warn users in our changelog. But as I said before it only happens when there is no other way. It is not a mandatory part of major releases.

**Comment by Bundy01** [comment]
Thanks again for answering my questions.
